src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,44 @@
 package core.basesyntax;
 
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
 public class SalaryInfo {
+    private static final String DATE_PATTERN = "dd.MM.yyyy";
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        SimpleDateFormat sdf = new SimpleDateFormat(DATE_PATTERN);
+        StringBuilder staffSalaryInformation = new StringBuilder("Report for period "
+                                             + dateFrom + " - " + dateTo);
```

Comment:
Use StringBuilder instead of String concatenation in loops. Also, replace \r\n with System.lineSeparator() for a platform-independent newline.